db/fixtures/classifications.yml

```diff
@@ -1,32 +1,32 @@
 --- 
```

Comment:
It looks like this file only contains whitespace changes. If that the case we need to remove it from the PR before we can merge.